src/datasources/checkorder.datasource.js

```diff
@@ -1,11 +1,12 @@
 import config from "../config";
+import isArray from "lodash/isArray";
 
 const { serviceRequester, url, ttl, prefix } = config.datasources.openorder;
 
 export async function load({ pid, pickupBranch, accessToken }, context) {
   const post = {
     pickUpAgencyId: pickupBranch,
-    pid: [pid],
+    pid: isArray(pid) ? pid : [pid],
```

Comment:
En mindre detalje, men hvis datasource servicen altid tager en liste af pider, skal vi så ikke kalde pid for pids? -> jeg ved godt at det allerede er sådan her i main - men jeg synes det giver bedre mening, så man ikke er i tvivl om at man skal sende en pid som en liste til datasourcen (og for at synliggøre at datasourcen kan tage pids som en liste).